core/app/models/spree/order.rb

```diff
@@ -663,6 +663,27 @@ def has_free_shipping?
               spree_promotion_actions: { type: 'Spree::Promotion::Actions::FreeShipping' }).exists?
     end
 
+    # all products are digital
```

Comment:
@MatthewKennedy can we move this into a separate module called `Spree::Order::Digital` eg. https://github.com/spree/spree/blob/main/core/app/models/spree/order/store_credit.rb

Order model is quite big already 😂